3_extracting_features/3.NF1_feature_extraction_env.yml

```diff
@@ -0,0 +1,7 @@
+name: 3.feature-extraction-NF1
+channels:
+  - conda-forge
+dependencies:
+  - conda-forge::python=3.8.13
+  - conda-forge::jupyter=1.0.0
+  - conda-forge::pandas=1.4.2
```

Comment:
```suggestion
  - conda-forge::pandas=1.4.2

```

Remember to address the red symbol :)

Reply:
@gwaybio I actually don't see a red symbol on my end when I don't have a extra line so that is why I keep missing it lol. I will try to memorize this!